### DISCUSSION TITLE
Diff flat data

### DISCUSSION THREAD

**Discussion by indifferentghost** [githubocto/flat-action#90] [Q&A]
https://github.com/githubocto/flat/discussions/70

Just kinda wanted to bring the idea of diffing flat data back.
I know it could be done in a post processing script.
I'm interested in seeing examples.

**Comment by chapmanjacobd** [comment]
Here are some options:

- https://github.com/simonw/git-history/
- https://github.com/dandavison/delta
- https://github.com/chapmanjacobd/library#tabular-data-subcommands
